schema/JF_TRAN_PAC_MEMO.json

```diff
@@ -67,15 +68,17 @@
       "description": "",
       "type": "string",
       "minLength": 0,
-      "maxLength": 12,
-      "pattern": "^[ -~]{0,12}$",
-      "examples": ["PAC_JF_MEM"],
+      "maxLength": 32,
+      "pattern": "^[ -~]{0,32}$",
+      "examples": [
+        "JF_TRANSFER_PAC_MEMOS"
+      ],
       "fec_spec": {
```

Comment:
Since this field can only have one value, the "const" schema directive can be used and assign the value to "JF_TRAN_PAC_MEMO". See here for an example: https://github.com/fecgov/fecfile-validate/blob/develop/schema/INDV_REC.json#L64-L78

Also, no "S" on JF_TRANSFER_PAC_MEMO on line 74

Reply:
Feedback incorporated